faker/providers/internet/__init__.py

```diff
@@ -3,10 +3,18 @@
 from __future__ import unicode_literals
 from .. import BaseProvider
 
+from ipaddress import ip_address, ip_network, IPV4LENGTH, IPV6LENGTH
+
 from faker.generator import random
 from faker.providers.lorem.la import Provider as Lorem
 from faker.utils.decorators import slugify, slugify_unicode
 
+try:
+    unicode
+except NameError:  # pragma: no cover
+    unicode = str  # pylint: disable=W0622,C0103
```

Comment:
Yes, was removed in https://github.com/kdeldycke/faker/commit/76720d7c35c436ba27638d9bfc25fb9e883ba0a7 . Just fixed that.